BoxContentSDK/BoxContentSDKTests/BOXCollaborationCreateRequestTests.m

```diff
@@ -21,7 +21,8 @@ @implementation BOXCollaborationCreateRequestTests
 - (void)test_that_request_with_user_id_has_expected_URLRequest
```

Comment:
we should add testing coverage to the `BOXCollaborationCreateRequest` created via the new file specific initializers